.changeset/nasty-cycles-decide.md

```diff
@@ -0,0 +1,5 @@
+---
+"@cloudflare/wrangler-devtools": patch
+---
+
+Add documentation on how to test with Playground preview in CI
```

Comment:
Updating readme itself shouldn't require a changeset, unless you wanna re-deploy it for some reason?

Reply:
Yeah, I wasn't sure. But that's a good point - I'll remove.